From 7ba5c150c993d8dc37851483fdf55f66ddfed0f6 Mon Sep 17 00:00:00 2001
From: Abel Buechner-Mihaljevic <Abel.Buechner@bosch-si.com>
Date: Thu, 9 Jan 2020 13:55:47 +0100
Subject: [PATCH] [#1696] Fix link to documentation for building images in
 readme of tests.

The readme file in the tests module points now to the section in the developer
guide that explains how to build the docker images from source.

Signed-off-by: Abel Buechner-Mihaljevic <Abel.Buechner@bosch-si.com>
---
 tests/readme.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/readme.md b/tests/readme.md
index d9b2cb052d..50e0aab57e 100644
--- a/tests/readme.md
+++ b/tests/readme.md
@@ -7,7 +7,7 @@ This module contains integration tests for Hono. The tests are executed against
 In order to run the tests you will need the following:
 
 * a working *Docker Engine* installation (either local or on a separate host)
-* the Docker images of the Hono components. See the [Getting Started guide on the project web site](https://www.eclipse.org/hono/getting-started/) for instructions on building the Docker images.
+* the Docker images of the Hono components. See the [Developer Guide on the project web site](https://www.eclipse.org/hono/docs/dev-guide/building_hono/) for instructions on building the Docker images.
 
 ## Running the Tests
 
